### PR TITLE
Website Updates

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
- Add weekly meeting minutes
- Advertise next two WG meetings
- Add latest HSF presentations to website

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
I will ask @jblomer about the proofer failure...

File: organization/_posts/2019/2019-05-23-coordination.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+## Activity and Working Group Updates
+
+### Detector Simulation
+  - Topical meeting held on 8 May on compute accelerators.
+  - Meeting 12 June, Detector Geometry Description Tools:
+      - [<span class="underline">https://indico.cern.ch/event/816485/</span>](https://indico.cern.ch/event/816485/)
+      - Please send us suggestions for talks and speakers\!
+
+### Reconstruction and Software Triggers
+  - Meeting yesterday on Atlas/CMS trigger systems/operations based on
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-05-23-coordination.md]: Better write "ATLAS".

File: organization/_posts/2019/2019-05-23-coordination.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+## Workshops
+
+### Latin American HSF Workshop
+  - Instituto Politécnico Nacional, Ciudad de México, Nov. 20-22 2019,
+    under the “auspices” of HSF.
+  - Mostly a gathering of the HEP Latin American community interested
+    in S&C with the goal to meet each other and discuss opportunities
+    for collaboration within the region and with the international
+    community.
+  - The scientific program is in the works.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-05-23-coordination.md]: "programme" as I believe we write in UK English.

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - fixed those little points.

**Comment by graeme-a-stewart** [comment]
Removing expired link as well

**Pull request merged**